src/ChooseThemeComponent/ChooseTheme.js

```diff
@@ -0,0 +1,119 @@
+import './ChooseTheme.css';
```

Comment:
In React.js importing css has some flaws. It will work initially but later when the components grow this will become problematic. Please refer to the link below for better CSS import approaches. I would highly recommend the "*.module.css" approach

https://medium.com/@ralph1786/using-css-modules-in-react-app-c2079eadbb87

Reply:
Sounds good! I just adjusted all the CSS components to fit the module format.